pkg/agent/sysadvisor/plugin/qosaware/resource/cpu/region/region_base.go

```diff
@@ -132,7 +132,6 @@ func (r *QoSRegionBase) Clear() {
 	r.Lock()
 	defer r.Unlock()
 
-	r.bindingNumas = machine.NewCPUSet()
```

Comment:
how QoSRegionBase matches with `not clear binding numas for dedicated cores`

Reply:
fixed